samza-sql/src/main/java/org/apache/samza/sql/interfaces/UdfMetadata.java

```diff
@@ -45,6 +46,8 @@ public UdfMetadata(String name, String description, Method udfMethod, Config udf
       SamzaSqlFieldType returnType, boolean disableArgCheck) {
     // Udfs are case insensitive
     this.name = name.toUpperCase();
+    // Let's also store the original name for display purposes.
+    this.displayName = name;
```

Comment:
This POJO already has hashcode and equals implemented in it. Are we planning to account for this new field in them?

Reply:
It should be ok not to account for the new field as name is already accounted for and we expect udfs to be anyways case insensitive.